src/instructlab/model/evaluate.py

```diff
@@ -424,6 +432,7 @@ def evaluate(
     gpus,
     merge_system_user_message,
     backend,
+    device,
```

Comment:
I assume this needs to be passed to get_evaluator and also passed to the additional instantiation of MMLUBranchEvaluator below?

Reply:
What Dan said

Please also add type annotation: `device: str,`.